### PR TITLE
EZSPA-793 - Create an airgap utility

### PR DESCRIPTION
**Pull request by mayankgarg23**

### CODE REVIEW

**Review by apurwaj2 — CHANGES_REQUESTED** [review]
The download image tool should still exist. It's a quick step to download some GCR image quickly.  The airgap utility task is to get a tar file of all the necessary spark images used by us.

File: tools/fetch_tag/fetch_tag.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        self.chart_files = dict()
+        self.example_files = dict()
+
+        self.re_pattern = re.compile("([a-zA-Z_-]+-\\d+.\\d+.*):(.+)")
+
+        self.charts_dir = os.path.abspath(os.path.join(cur_dir, "../../charts"))
+        self.examples_dir = os.path.abspath(os.path.join(cur_dir, "../../examples"))
+        self.imgtxt_file = os.path.abspath(os.path.join(cur_dir, "../imagelist.txt"))
+    def run(self):
+        self.get_tags_from_charts_yaml(self.charts_dir, self.chart_files, ["livy-0.8.0", "metastore-3.1.1", "mysql-operator", "spark-3.1.1", "spark-hs-3.1.1"])
```
Comment (apurwaj2) [inline tools/fetch_tag/fetch_tag.py:25]: Why have you included these charts?

**Review by apurwaj2 — CHANGES_REQUESTED** [review]

File: tools/fetch_tag/fetch_tag.py (hunk 1 above)
Comment (mayankgarg23) [reply tools/fetch_tag/fetch_tag.py:25]: these are the ones which needs to be ignored

File: tools/download_image/download-images.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
```
Comment (apurwaj2) [inline tools/download_image/download-images.sh:1]: Rename this file to download all spark images or let's not confuse user with similar name scripts.

File: tools/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Script to fetch image tags and download these images from GCR repository on local setup
```
Comment (apurwaj2) [inline tools/README.md:1]: This readme is applicable to both the scripts? Previous single image download as well as image download for all spark images?

**Review by apurwaj2 — CHANGES_REQUESTED** [review]

File: tools/download_image/download-images.sh (hunk 2 above)
Comment (mayankgarg23) [reply tools/download_image/download-images.sh]: updated script name to download-single-image.sh and download-all-images.sh

File: tools/README.md (hunk 3 above)
Comment (mayankgarg23) [reply tools/README.md]: There are different README.md files, one at tools/ directory level (for downloading all images) and other at donwload_image/ directory level (for downloading a single image).

**Review by apurwaj2 — APPROVED** [review]
lgtm

**Pull request merged**